test/search-filters.test.js

```diff
@@ -739,3 +889,18 @@ testWithServer('Should accept multiple image_licences', (t, ctx) => {
     t.end();
   });
 });
+
+testWithServer('Should not accept array of multiple image_licences as json', (t, ctx) => {
+  t.plan(1);
```

Comment:
Possibly not, but [Alan suggested that we should](https://github.com/TheScienceMuseum/collectionsonline/pull/28#issuecomment-226163479)